endpoints/openrtb2/amp_auction.go

```diff
@@ -541,6 +541,7 @@ func (deps *endpointDeps) overrideWithParams(ampParams amp.Params, req *openrtb2
 	if req.Site == nil {
 		req.Site = &openrtb2.Site{}
 	}
+	var errors []error
```

Comment:
For my own understanding: Why were there changes made to error handling? Did it have something to do with FPD?

Reply:
This change was requested in this comment: https://github.com/prebid/prebid-server/issues/1727#issuecomment-1488952244
It was a fatal error before. But because we need to ignore an error I modified it to warning and added to the list of existing errors. So if there were merge errors here in `setTargeting` we just ignore it and continue `overrideWithParams`.